packages/react-async/README.md

```diff
@@ -0,0 +1,147 @@
+# `@shopify/react-async`
+
+[![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Freact-async.svg)](https://badge.fury.io/js/%40shopify%2Freact-async.svg) [![npm bundle size (minified + gzip)](https://img.shields.io/bundlephobia/minzip/@shopify/react-async.svg)](https://img.shields.io/bundlephobia/minzip/@shopify/react-async.svg)
+
+Tools for creating powerful, asynchronously-loaded React components.
+
+## Installation
+
+```bash
+$ yarn add @shopify/react-async
+```
+
+## Usage
+
+### `createAsyncComponent()`
+
+`createAsyncComponent` is a function for creating components that are loaded asynchronously on initial mount. However, the resulting component does more than just help you split up your application along component lines; it also supports customized rendering for loading, and creates additional components for smartly preloading or preloading the component’s bundle. Best of all, in conjunction with the Babel and Webpack plugins provided by [`@shopify/async`](https://github.com/Shopify/quilt/tree/master/packages/async), you can easily extract the bundles needed to render your application during server side rendering.
+
+To start, import the `createAsyncComponent` function. The simplest use of this function requires just a `load` function, which returns a promise for a component:
+
+```tsx
+import {createAsyncComponent} from '@shopify/react-async';
+
+const MyComponent = createAsyncComponent({
+  load: () => import('./MyComponent'),
+});
+```
+
+This function returns a component that accepts the same props as the original one.
+
+`createAsyncComponent` also adds a few static members that are themselves components: `Preload`, `Prefetch`, and `KeepFresh`.
+
+```tsx
+const MyComponent = createAsyncComponent({
+  load: () => import('./MyComponent'),
+});
+
+// All of these are available:
+<MyComponent />
+<MyComponent.Prefetch />
+<MyComponent.Preload />
+<MyComponent.KeepFresh />
+```
+
+By default, `Preload`, `Prefetch`, and `KeepFresh` all simply prefetch the bundle for the component in the background. However, you can provide additional markup to render during these phases with the `renderPreload`, `renderPrefetch`, and `renderKeepFresh` options to `createAsyncComponent`:
```

Comment:
How do these phases work? Is there an example async lifecycle we could provide?

Reply:
not sure what you mean by phases, but the docs below show what we recommend using each one for.

Reply:
Neither am I. You mention the word phases  in the readme.

Reply:
Gotcha, thanks. Removed that part because it wasn't needed 👍